File: python/leap/leap.py
def leap_year(year):
    if (evenly_divisible(year, 100) and not evenly_divisible(year, 400)):
        return False

    if (evenly_divisible(year, 4)):
        return True
    
    return False


def evenly_divisible(year, number):
    return year % number == 0

File: dart/leap/lib/leap.dart
class Leap {
  // Put your code here
  bool leapYear(int year) {
    if (year % 100 == 0 && year % 400 == 0) {
      return true;
    }

    if (year % 4 == 0) {
      return true;
    }

    return false;
  }
}
